ols/constants.py

```diff
@@ -221,3 +221,6 @@ class GenericLLMParameters:
 # Response streaming media types
 MEDIA_TYPE_TEXT = "text/plain"
 MEDIA_TYPE_JSON = "application/json"
+
+# default value for token when no token is provided
+NO_TOKEN = ""
```

Comment:
NO_USER_TOKEN or something like this? I mean, `token` has different meaning in NLP world